docs/design/SUMMARY_API.md

```
@@ -1,155 +1,50 @@
----
-
-## 概要
-
-ダッシュボードのグラフ・合計額を Stimulus + fetch で取得するための JSON API。
-
----
+ダッシュボード（#48）が消費する月次集計 JSON を返す読み取り専用エンドポイント。
 
 ## エンドポイント
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)